gc/base/ObjectScanner.hpp

```diff
@@ -61,7 +68,8 @@ class GC_ObjectScanner : public MM_BaseVirtual
 	uintptr_t _leafMap;						/**< Bit map of reference slots in object that refernce leaf objects */
 #endif /* defined(OMR_GC_LEAF_BITS) */
 	fomrobject_t *_scanPtr;					/**< Pointer to base of object slots mapped by current _scanMap */
-	GC_SlotObject _slotObject;				/**< Create own SlotObject class to provide output */
+	GC_SlotObject *_slotObjectPtr;			/**< Points to effective slot object for receiving refernce slots */
```

Comment:
Can you fix the spacing on the comment?

Reply:
I think what you are seeing is an artifact of html layout (tab size). It looks fine in eclipse:

![image](https://user-images.githubusercontent.com/18146124/41176126-51914f1e-6b2d-11e8-8644-b11c5f51ec67.png)